chain/client/src/client.rs

```diff
@@ -65,6 +68,16 @@ pub struct ClientActor {
     num_blocks_processed: u64,
     /// Total number of transactions processed.
     num_tx_processed: u64,
+
+
+}
+
+fn wait_until_genesis(genesis_time: &DateTime<utc>) -> Result<(), OutOfRangeError> {
+        let now : DateTime<Utc> = Utc::now();
+        let duration = now.signed_duration_since(*genesis_time);
+        let std_duration = duration.to_std()?;
+        thread::sleep(std_duration);
```

Comment:
let's actually add logging here target: "info", "Waiting for genesis: " and pretty printed time.

Reply:
what logger has to be used? There are init_integration_logger ,  init_test_logger, env_logger available.

Reply:
you don't need to use those, that is for setting up logging, done once per program -> just use `info!(target: "info", "...")`